infra/charts/feast-feature-server/templates/deployment.yaml

```diff
@@ -52,7 +57,8 @@ spec:
             - "serve"
             - "-h"
             - "0.0.0.0"
-            {{- end }}
+            {{- end }} 
+          command: ["feast", "serve", "-h", "0.0.0.0"]
```

Comment:
this should likely turn into something like -
```yaml
{{- if .Values.metrics }}
command: ["feast", "serve", "--metrics", "-h", "0.0.0.0"]
{{- else }}
command: ["feast", "serve", "-h", "0.0.0.0"]
```

Reply:
@tsisodia10 if the `--metrics` flag is NOT specified, prom and telemetry should NOT be enabled. Metrics should only enabled if the flag is used. Based on this understanding, we would need this helm check. Is this not currently how the cli flag is functioning?

Reply:
i don't see it being passed anywhere in the helm charts... how it is currently being enabled?

Reply:
@dmartinol related thread